collectors/python.d.plugin/python.d.conf

```diff
@@ -71,7 +71,7 @@ logind: no
 
 # nginx_log has been replaced by web_log
 nginx_log: no
-# ntpd: yes
+nvidia_smi: no
```

Comment:
Why are you removing `# ntpd: yes`?

```suggestion
# ntpd: yes
nvidia_smi: no
```

Reply:
Because there are two of them.
```
# ntpd: yes

# nginx_log has been replaced by web_log
nginx_log: no
nvidia_smi: no
# ovpn_status_log: yes
```

Reply:
Removing the duplicate entry should be a separate PR from this one, as it's an unrelated change.

Reply:
in #4460